packages/website/src/components/Modals/RecruitmentModal.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import {
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+} from '@coderscamp/ui/components/Modal';
+import { Typography } from '@coderscamp/ui/components/Typography';
+
+import { forMentors } from './modal.data';
+import { ModalForm } from './ModalForm';
+
+type RecruitmentModalProps = {
+  isOpen: boolean;
+  onClose: () => void;
+  modalText?: {
+    header: string;
+    body: string;
+    footer: string;
+  };
+};
+
+export const RecruitmentModal = ({ isOpen, onClose, modalText = forMentors }: RecruitmentModalProps) => {
+  return (
+    <>
```

Comment:
❓ Why do we need this fragment?

Reply:
I thought I would have to give these props to the modal but I solved it a little different now.